src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,24 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService)
+                injector.getInstance(DriverService.class);
+        Driver driver = new Driver(4L,"Yuriy", "sm002244");
```

Comment:
Do we need to set id here, isn't it AUTO_INCREMENT?

Reply:
In README class Driver already was with field id. That`s why I create constructor with id. But in my table drivers in BD I use AUTO_INCREMENT